cms/src/components/content/ContentOverview.spec.ts

```diff
@@ -126,6 +126,32 @@ describe("ContentOverview.vue", () => {
         });
     });
 
+    it("should display filter options and inputs", async () => {
```

Comment:
I'm missing some tests here for the functionality.

Reply:
Added tests